src/h5web/providers/mock/utils.ts

```diff
@@ -192,6 +192,10 @@ export function makeExternalLink(
 /* ----------------- */
 /* ----- NEXUS ----- */
 
+function makeNxAxesAttr(axes: string[]): HDF5Attribute {
+  return makeAttr('axes', makeSimpleShape([axes.length]), stringType, axes);
+}
+
```

Comment:
I'm shuffling some functions around and I'm refactoring `makeNxDataGroup` so it calls `makeNxGroup` internally instead of `makeGroup`. The former adds the `NX_class` attribute. It's not much, but it feels more logical and consistent with other mock data functions.